Justfile

```diff
@@ -159,6 +159,15 @@ check-lychee:
     @echo {{ if env("GITHUB_TOKEN", "") != "" { "" } \
              else { "Note: 'Too Many Requests' errors are allowed here but not in CI, set GITHUB_TOKEN to check them" } }}
 
+# check tools/protocol-schema-check/res/protocol_schema.toml
+check-protocol-schema:
+    rustup toolchain install nightly
+    rustup target add wasm32-unknown-unknown --toolchain nightly
```

Comment:
I was going to say that this is not the right place to install toolchains or add targets, and that it ought to be done by the calling environment, but then I see that `cargo-udeps` job does the same. So lets leave as-is for now and maybe I'll get around fixing every instance of this later...